lib/__tests__/harvest.test.ts

```diff
@@ -124,16 +124,38 @@ describe("Integration tests", () => {
       it("should retrieve a queryID on page load", async () => {
         expect(data).toHaveProperty("queryID");
       });
-      it("should generate an anonymous userToken on init", async () => {
-        const userToken = await page.evaluate(() => {
-          return new Promise((resolve, reject) =>
-            window.aa("getUserToken", null, (err, res) => {
-              if (err) return reject(err);
-              resolve(res);
-            })
-          );
-        });
+      it("should generate an anonymous userToken on init and store it in a cookie", async () => {
+        const userToken = await page.evaluate(
+          () =>
+            new Promise((resolve, reject) =>
+              window.aa("getUserToken", null, (err, res) => {
+                if (err) return reject(err);
+                resolve(res);
+              })
+            )
+        );
+
+        const cookies = await page.cookies();
+        const algoliaCookie = cookies.find(
+          cookie => (cookie.name = "_ALGOLIA")
```

Comment:
Not sure to understand this assignment. Did you mean this?

```suggestion
          cookie => cookie.name === "_ALGOLIA"
```

Reply:
yes absolutely, thanks for catching the typo